settings/NoticePolicy/NoticePolicyForm.js

```diff
@@ -45,12 +50,28 @@ class NoticePolicyForm extends React.Component {
     showDeleteConfirmation: false,
     sections: {
       general: true,
-      loanNotices: false,
+      loanNotices: true,
       feeFineNotices: false,
       requestNotices: false,
     },
   };
 
+  getTemplates = (noticeCategory) => {
```

Comment:
It looks like `getTemplates` exists in both `NoticePolicyForm` and `NoticePolicyDetail`. Perhaps it would be valuable to extract it from the components into a `util` file so they can be reused.